capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/camera/photo/PhotoCropModifier.java

```diff
@@ -0,0 +1,91 @@
+package net.gini.android.capture.internal.camera.photo;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Matrix;
+import android.graphics.Rect;
+import android.util.DisplayMetrics;
+
+import java.io.ByteArrayOutputStream;
+
+public class PhotoCropModifier implements PhotoModifier {
+
+    private final Photo mPhoto;
+    private final int[] mScreenSize;
+    private final Rect aRect;
+
+    public PhotoCropModifier(Photo mPhoto, int[] mScreenSize, Rect aRect) {
+        this.mPhoto = mPhoto;
+        this.mScreenSize = mScreenSize;
+        this.aRect = aRect;
+    }
+
+    @Override
+    public void modify() {
+
+        if (mPhoto.getData() == null)
+            return;
+
+        synchronized (aRect) {
+            byte[] originalBytes = mPhoto.getData();
+            try {
+                float rotation = 0;
+
+                switch (mPhoto.getRotationForDisplay() % 360) {
+                    case 90:
+                        rotation = 90;
+                        break;
+                    case 180:
+                        rotation = 180;
+                        break;
+                    case 270:
+                        rotation = 270;
+                        break;
+                    default:
+                        break;
+                }
```

Comment:
[Suggestion] Can this be simplified to `float rotation = mPhoto.getRotationForDisplay() % 360`?

Reply:
Yeah true. I used to set `finalRotation()` but as the matrix has inverse makes this obsolete.